test/ClangImporter/nsenum-with-unavailable-typedef.swift

```diff
@@ -0,0 +1,60 @@
+// RUN: split-file %s %t
+// RUN: %target-swift-frontend -typecheck -verify %t/main.swift -I %t/include -module-name main
+// RUN: %target-swift-frontend -typecheck -verify %t/main.swift -I %t/include -module-name main -cxx-interoperability-mode=default
+// REQUIRES: objc_interop
+
+//--- include/module.modulemap
+module ObjCModule {
+    header "header.h"
+}
+
+//--- include/header.h
```

Comment:
Should we have this test somewhere around `test/Interop/Cxx/objc-correctness/Inputs/CFOptions.h`?

Reply:
I put these here because these tests are not C++-specific. The `test/Interop/C/` subdirectory didn't really seem to have much in them, and besides, this isn't really C-specific either. I don't feel strongly about this, though.